core/store/src/db.rs

```diff
@@ -830,7 +832,22 @@ fn rocksdb_column_options(col: DBCol) -> Options {
     opts
 }
 
+// Number of RocksDB instances in the process.
+pub static ROCKSDB_INSTANCES_COUNTER: Lazy<(Mutex<usize>, Condvar)> =
```

Comment:
Can this be at least `pub(crate)` ? Exposing global state to the world isn't great.